pkg/protobuf/deploy/deploy.proto

```diff
@@ -37,8 +37,9 @@ message ListResponse {
     message Deploy {
         string revision = 1;
         string description = 2;
-        int64 age  = 3;
+        reserved 3;
```

Comment:
mixed fellings

Reply:
Another option is rename to `OBSOLETE_age` but not seems good =/